extensions/panache/rest-data-panache/runtime/src/main/java/io/quarkus/rest/data/panache/ResourceProperties.java

```diff
@@ -0,0 +1,31 @@
+package io.quarkus.rest.data.panache;
+
+import static java.lang.annotation.ElementType.TYPE;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+@Documented
+@Retention(RUNTIME)
+@Target({ TYPE })
+public @interface ResourceProperties {
```

Comment:
Could probably do with javadoc to mention this annotation is not mandatory unless you want to override the defaults.

Reply:
Done